diff --git a/node_modules/@hrwg/vue-marked/src/VueMarked.vue b/node_modules/@hrwg/vue-marked/src/VueMarked.vue
index ea6dbc0..10be550 100644
--- a/node_modules/@hrwg/vue-marked/src/VueMarked.vue
+++ b/node_modules/@hrwg/vue-marked/src/VueMarked.vue
@@ -1,6 +1,5 @@
 <template>
     <div v-html="markup">
-        <slot></slot>
     </div>
 </template>
 
@@ -13,28 +12,16 @@
                 type: Function,
                 default: marked,
             },
-        },
-
-        data() {
-            return {
-                source: '',
-            };
+            source: {
+                type: String,
+                default: ''
+            }
         },
 
         computed: {
             markup() {
                 return this.marked(this.source);
             },
-        },
-
-        beforeMount() {
-            if (!this.$slots.default) {
-                return;
-            }
-
-            this.source = this.$slots.default.reduce((source, slot) => {
-                return `${source}${slot.text.trim()}`;
-            }, this.source);
-        },
+        }
     };
 </script>
diff --git a/node_modules/@hrwg/vue-marked/src/index.d.ts b/node_modules/@hrwg/vue-marked/src/index.d.ts
new file mode 100644
index 0000000..ece4984
--- /dev/null
+++ b/node_modules/@hrwg/vue-marked/src/index.d.ts
@@ -0,0 +1,4 @@
+import { Component } from "@vue/runtime-core"
+
+declare const VueMarked: Component
+export default VueMarked; 
